packages/near-membrane-base/src/instrumentation.ts

```diff
@@ -0,0 +1,10 @@
+export interface Activity {
+    start: () => Activity;
+    stop: () => Activity;
+}
+
+export interface Instrumentation {
+    activityBeacon: (activityName: string) => Activity;
+    errorBeacon: (sandboxKey: string, e: Error) => void;
+    logBeacon: (sandboxKey: string, message: string) => void;
+}
```

Comment:
The same interfaces are needed in Locker project as well. The code in Locker will need to be changed to reference the interfaces here.